db/r2yara

```diff
@@ -1,6 +1,6 @@
 R2PM_BEGIN
 
-R2PM_GIT "https://github.com/radareorg/radare2-extras"
+R2PM_GIT "https://github.com/Yara-Rules/r2yara.git"
```

Comment:
Thats the wrong link

Reply:
Ah sorry I was also testing something else.